packages/sdk/src/evm/common/currency.ts

```diff
@@ -192,3 +192,22 @@ export async function normalizeAmount(
   const decimals = await contractWrapper.readContract.decimals();
   return utils.parseUnits(AmountSchema.parse(amount), decimals);
 }
+
+export function toEther(amount: Amount): string {
+  return utils.formatEther(AmountSchema.parse(amount));
+}
+
+export function toWei(amount: BigNumber | string): BigNumber {
+  return utils.parseEther(AmountSchema.parse(amount));
+}
+
+export function toUnits(amount: Amount, decimals: number): BigNumber {
+  return utils.parseUnits(AmountSchema.parse(amount), decimals);
+}
+
+export function formatUnits(
```

Comment:
how about calling this `toDisplayValue` instead so its clear what the intent is?

Reply:
Yes much better